matchbox_demo/src/main.rs

```diff
@@ -27,32 +27,34 @@ fn main() {
     info!("{:?}", args);
 
     let mut app = App::new();
-    app.insert_resource(Msaa { samples: 4 })
-        .insert_resource(ClearColor(SKY_COLOR))
-        .add_plugins(DefaultPlugins)
-        // Some of our systems need the query parameters
-        .insert_resource(args)
-        // Make sure something polls the message tasks regularly
-        .add_plugin(GGRSPlugin)
-        // define frequency of game logic update
+
+    GGRSPlugin::<GGRSConfig>::new()
+        // define frequency of rollback game logic update
         .with_update_frequency(FPS)
-        // define system that represents your inputs as a byte vector, so GGRS can send the inputs around
+        // define system that returns inputs given a player handle, so GGRS can send the inputs around
         .with_input_system(input)
-        // register components that will be loaded/saved
+        // register types of compontents AND resources you want to be rolled back
         .register_rollback_type::<Transform>()
         .register_rollback_type::<Velocity>()
-        // you can also register resources
-        .insert_resource(FrameCount { frame: 0 })
         .register_rollback_type::<FrameCount>()
         // these systems will be executed as part of the advance frame update
         .with_rollback_schedule(
             Schedule::default().with_stage(
-                "rollback_default",
-                SystemStage::single_threaded()
+                ROLLBACK_DEFAULT,
+                SystemStage::parallel()
                     .with_system(move_cube_system)
                     .with_system(increase_frame_system),
             ),
         )
+        // make it happen in the bevy app
+        .build(&mut app);
+
+    app.insert_resource(Msaa { samples: 4 })
+        .insert_resource(ClearColor(SKY_COLOR))
+        .add_plugins(DefaultPlugins)
+        // Some of our systems need the query parameters
+        .insert_resource(args)
+        .insert_resource(FrameCount::default())
```

Comment:
```suggestion
        .init_resource::<FrameCount>()
```

Reply:
Didn't know that worked. Made the change.